src/components/videoPage/VideoDescription.tsx

```diff
@@ -23,13 +23,17 @@ export const VideoDescription = ({ video }: Props) => {
         ))}
       </div>
 
-      <Typography.Body size="small" className="lg:mt-4 text-gray-800">
-        {video?.description}
-      </Typography.Body>
+      <div className="mt-4">
```

Comment:
What do you think about using a semantic [`<section>`](https://www.w3schools.com/html/html5_semantic_elements.asp) element instead? We should also be able to use a typography heading instead of a body and a `<br />` element.

Reply:
I like this idea thanks